### ISSUE TITLE
create google analytics if you havent

### ISSUE THREAD

**Issue by ran-codes** [Lfrueh/SSP#5]
(just saw your app on linkedin) ... looks great. I love the branding !

btw... in case your app goes viral (there is a chance); you should add the google analytics to track it (if you haven't already)

- see https://shiny.posit.co/r/articles/build/google-analytics/#:~:text=To%20add%20Google%20Analytics%20to%20a%20Shiny%20app%3A,section%20of%20your%20app%27s%20DOM..
- you can skip event trackers. (page visits is enough to get a sense of traffic).

**Comment by ran-codes** [comment]
in the past I've made apps and thought i won't get traffic so no need to add a traffic tracker ... which then got a lot of traffic and I regretted it (so speaking from experience).  😢

**Comment by Lfrueh** [comment]
Good call. I realized this might be a good thing to do with all my research dissemination products!

Thank you!

From: Ran Li ***@***.***>
Reply-To: Lfrueh/SSP ***@***.***>
Date: Tuesday, February 27, 2024 at 8:47 PM
To: Lfrueh/SSP ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [Lfrueh/SSP] create google analytics if you havent (Issue #5)


in the past I've made apps and thought i won't get traffic so no need to add a traffic tracker ... which then got a lot of traffic and I regretted it (so speaking from experience). 😢
